packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/common.ts

```diff
@@ -74,4 +189,11 @@ export interface CommonDestinationProps {
    * @default [] - no data transformation will occur.
    */
   readonly processors?: firehose.IDataProcessor[];
+
+  /**
+   * The configuration for backing up source records to S3.
+   *
+   * @default - source records will not be backed up to S3.
+   */
+  readonly backupConfiguration?: S3BackupDestinationProps;
```

Comment:
I don't love this name. How about `destinationBackup`? Or `destinationS3Backup`? That last one works nicely iff you apply my suggestion from above to call the type `DestinationS3BackupProps`.

Reply:
I think the "destination" is redundant here. What about `s3Backup` or `s3BucketBackup`?

Reply:
I like `s3Backup`.

Reply:
`backup` is probably also acceptable. Your call.